rpc/src/rpc.rs

```diff
@@ -1791,8 +1791,10 @@ impl JsonRpcRequestProcessor {
                             }
                         }
                     }
+                    Err(StorageError::SignatureNotFound) => {}
```

Comment:
A failure to find the `before` or `until` signatures continues to behave like the old code: skip and move on.